test/dtypes/test_affine_quantized_float.py

```diff
@@ -288,7 +290,182 @@ def test_fp8_weight_dimension_warning(self):
             )
 
 
+# copied from https://github.com/Dao-AILab/flash-attention/blob/1feb711f46563960fc10a8e659c93c300619504b/tests/test_util.py#L185
+
+
+def construct_local_mask(
```

Comment:
I have variants of these that dont require einops
https://github.com/pytorch/pytorch/blob/f9af86de0138646540980148a781b7709f03c998/test/test_transformers.py#L2381

Reply:
removed this since we are matching original floating point output now